benchmark.py

```diff
@@ -15,7 +15,7 @@
 
 def main():
 
```

Comment:
```
config = tf.compat.v1.ConfigProto()
config.gpu_options.allow_growth = True
with tf.compat.v1.Session(config=config) as sess:
```
This will allow RTX card to work. See https://github.com/tensorflow/tensorflow/issues/24496.